en/mozillavpn.xliff

```diff
@@ -0,0 +1,556 @@
+      <context-group purpose="location"><context context-type="sourcefile">../src/ui/views/ViewUpdate.qml</context><context context-type="linenumber">194</context></context-group>
+    </trans-unit>
+  </body></file>
+  <file datatype="plaintext" original="../src/ui/views/ViewOnboarding.qml" source-language="en" target-language="en-US"><body>
+    <trans-unit id="vpn.onboarding.skip">
+      <source xml:space="preserve">Skip</source>
+      <context-group purpose="location"><context context-type="linenumber">26</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.onboarding.next">
+      <source xml:space="preserve">Next</source>
+      <context-group purpose="location"><context context-type="linenumber">54</context></context-group>
+    </trans-unit>
+  </body></file>
+  <file datatype="plaintext" original="../src/ui/views/ViewUpdate.qml" source-language="en" target-language="en-US"><body>
+    <trans-unit id="vpn.updates.updateRecomended">
+      <source xml:space="preserve">Update recommended</source>
+      <context-group purpose="location"><context context-type="linenumber">29</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.updates.updateRecomended.description">
+      <source xml:space="preserve">Please update the app before you\ncontinue to use the VPN</source>
+      <context-group purpose="location"><context context-type="linenumber">31</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.updates.updateRequired">
+      <source xml:space="preserve">Update Required</source>
+      <context-group purpose="location"><context context-type="linenumber">68</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.updates.updateRequire.reason">
+      <source xml:space="preserve">We detected and fixed a serious bug.\nYou must update your app.</source>
+      <context-group purpose="location"><context context-type="linenumber">70</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.updates.updateConnectionInsecureWarning">
+      <source xml:space="preserve">Your connection will not be secure while you update.</source>
+      <context-group purpose="location"><context context-type="linenumber">183</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.updates.notNow">
+      <source xml:space="preserve">Not now</source>
+      <context-group purpose="location"><context context-type="linenumber">207</context></context-group>
+    </trans-unit>
+  </body></file>
+  <file datatype="cpp" original="../src/connectiondataholder.cpp" source-language="en" target-language="en-US"><body>
+    <trans-unit id="vpn.connectionInfo.unknown">
+      <source xml:space="preserve">Unknown</source>
+      <context-group purpose="location"><context context-type="linenumber">29</context></context-group>
+      <note annotates="source" from="developer">Context - "The current ip-address is: unknown"</note>
```

Comment:
> This refers to the current IP address.

Is this used in the `vpn.connectionInfo.ip` string? If so, I would also add 

> This refers to the current IP address, i.e. "IP: Unknown".

That's important in order to understand if the translation should start lowercase or uppercase.